drivers/include/ili9341.h

```diff
@@ -98,6 +102,9 @@ typedef struct {
  * @brief   Device descriptor for a ili9341
  */
 typedef struct {
+#ifdef MODULE_DISP_DEV
```

Comment:
I'm not sure if this shouldn't be the other way around: if the `ili9341_disp_dev_t` should have a `disp_dev_t` as first member and a `ili9341_t` as second member?

Reply:
the `ili9341_disp_dev_t` struct doesn't exist. This scheme is inspired by netdev.

Reply:
ok